doc/adrs/191_add_diagram_elements_appearance_management.adoc

```diff
@@ -0,0 +1,27 @@
+= ADR-191 - Add the capability of customizing the appearance of diagram elements
+
+== Context
+
+The appearance of diagram elements is not customizable outside of what is defined in the specifier.
+
+== Decision
+
+End-users will be able to customize the appearance of diagram elements through the palette, these customizations will be persisted but can be erased to fall back to the specified appearance.
+
+== Backend
+
+A new field in nodes and edges will store their customized appearance, the field may be null when there is no customization.
+
+A new type of diagram events will enable the renderer to update the customized appearance of nodes and edges as they render.
```

Comment:
Why? There is already an event from graphical changes in the diagram that do not involve changes in the semantic data. We could update its name but I don't see why a second event would be necessary.

Reply:
I meant a type of IDiagramEvent similarly to what has been done for hiding/fading elements for example.